pkg/repository/v1/sqlcv1/log_line.sql

```diff
@@ -0,0 +1,29 @@
+-- name: InsertLogLine :copyfrom
+INSERT INTO v1_log_line (
+    tenant_id,
+    task_id,
+    task_inserted_at,
+    message,
+    metadata
+) VALUES (
+    $1,
+    $2,
+    $3,
+    $4,
+    $5
+);
+
+-- name: ListLogLines :many
+SELECT
+    *
+FROM
+    v1_log_line l
+WHERE
+    l.tenant_id = @tenantId::uuid
+    AND l.task_id = @taskId::bigint
+    AND l.task_inserted_at = @taskInsertedAt::timestamptz
+    AND (sqlc.narg('search')::text IS NULL OR l.message LIKE concat('%', sqlc.narg('search')::text, '%'))
```

Comment:
same qq re: `ilike` here as before - I think for log search `ilike` is a lot safer, would be odd to search for `foo` and miss `Foo`